kie-wb-smoke-tests/pom.xml

```diff
@@ -385,6 +391,27 @@
                 <groups>org.kie.smoke.wb.category.KieWbSeleniumSmoke</groups>
               </configuration>
             </plugin>
+            <plugin> <!-- Download & unzip IEDriverServer.exe - a binary required to run InternetExplorerDriver -->
+              <artifactId>maven-antrun-plugin</artifactId>
+              <executions>
+                <execution>
+                  <phase>pre-integration-test</phase>
+                  <goals>
+                    <goal>run</goal>
+                  </goals>
+                  <configuration>
+                    <target>
+                      <get src="http://selenium-release.storage.googleapis.com/2.48/IEDriverServer_x64_2.48.0.zip"
+                           dest="${java.io.tmpdir}"
+                           skipexisting="true"/>
+                      <unzip src="${java.io.tmpdir}/IEDriverServer_x64_2.48.0.zip"
+                             dest="${java.io.tmpdir}"
```

Comment:
I think that we should unzip this into target/ so that we can rely on "clean" builds. When we depend on some content in /tmp maven will not clean that up and we may end up with some weird bugs related to old version of the IE driver in the /tmp. In ideal case, we should download the zip file only once (and cache it) and unzip it with every clean build. This is what download-maven-plugin should be able to do.